packages/core/src/presets/aiostreams.ts

```
@@ -0,0 +1,141 @@
+      SUPPORTED_RESOURCES: [],
+    };
+  }
+
+  static async generateAddons(
+    userData: UserData,
+    options: Record<string, any>
+  ): Promise<Addon[]> {
+    if (!options.manifestUrl.endsWith('/manifest.json')) {
+      throw new Error('Invalid manifest URL');
+    }
+    return [this.generateAddon(userData, options)];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)